ystafdb/config_parser.py

```diff
@@ -1,13 +1,17 @@
+import os
 import json
-
+from pathlib import Path
 
 def get_config_data():
     providers = []
     datasets = []
     providersRequiredFields = set()
     datasetsRequiredFields = set()
 
-    with open('ystafdb/data/config.json') as json_file:
+    if not os.path.exists(Path("config.json")):
+        exit("No config.json file")
+
+    with open('config.json') as json_file:
```

Comment:
Don't copy the same path multiple locations, assign it to a variable and use the variable

Reply:
a46b58f resolves this issue